profile_library/signify/LCX002/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-12-11T02:55:33.655121",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "shelly,S3PM-001PCEU16",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.16.2:docker"
+  },
+  "name": "LCX002",
```

Comment:
This must be the fully marketed name. Will be shown in name column in the library viewer.
See https://library.powercalc.nl/ and filter on manufacturer signify for other examples.